components/event/CreateEvent/Steps/CreateEventStep3.tsx

```diff
@@ -1,8 +1,185 @@
-import { Box } from "@mui/material"
-import React from "react"
+import {
+  Box,
+  Checkbox,
+  FormControl,
+  FormControlLabel,
+  FormGroup,
+  FormHelperText,
+  FormLabel
+} from "@mui/material"
+import React, { useContext, useState } from "react"
+import FlatCard from "../../../UI/card/FlatCard"
+import ControlledStepperButtons from "../../../UI/navigation/ControlledStepperButtons"
+import { CreateEventContext } from "../../../../contexts/event/CreateEventContext"
+import { useFormik } from "formik"
+import {
+  Event,
+  TicketSupplySettings,
+  TicketTypeKey
+} from "../../../../interfaces/event/event.interface"
+import { TextFieldWrapper } from "../../../UI/textfield/TextFieldWrapper"
+import { CreateEventStep3Schema } from "../../../../schema/event/createEventStep3.schema"
 
 function CreateEventStep3() {
-  return <Box>CreateEventStep3</Box>
+  // Hooks
+  const {
+    event: currentEvent,
+    setEvent,
+    setActiveStep
+  } = useContext(CreateEventContext)
+  const { values, errors, touched, handleSubmit, setFieldValue } = useFormik({
+    initialValues: {
+      general: currentEvent.ticketSupply.general,
+      vip: currentEvent.ticketSupply.vip,
+      reservedSeat: currentEvent.ticketSupply.reservedSeat
+    },
+    validationSchema: CreateEventStep3Schema,
+    onSubmit: async (data) => {
+      console.log(data)
+      const _event: Event = {
+        ...currentEvent,
+        ticketSupply: data
+      }
+      setEvent(_event)
+    }
+  })
+
+  // States
+
+  const [ticketSupply, setTicketSupply] = useState<TicketSupplySettings[]>([
+    {
+      type: "general",
```

Comment:
I think I use 'GENERAL', 'RESERVED_SEATS', etc. Plz refer to my ticket type

Reply:
I will change this at once later, caz it's like this right now in the back end.